aws/env/variables.tf

```diff
@@ -0,0 +1,69 @@
+# Copyright 2021 Datastax LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Variables to pass into the aws terraform aws modules
+variable "name" {
+  description = "The name to give the new Kubernetes cluster resources."
+  type        = string
+}
+
+variable "environment" {
+  description = "Name of the environment where infrastrure being built."
+  type        = string
+}
+
+variable "resource_owner" {
+  description = "The name of the Project Owner"
+  type        = string
+  default     = "Datastax"
+}
+
+variable "region" {
+  description = "The aws region in which resources will be defined."
+  type        = string
+  default     = "us-east-1"
+}
+
+variable "instance_type" {
+  description = "Type of instance to be used in the k8ssandra."
+  type        = string
+  default     = "t2.medium"
```

Comment:
Should this be updated to a more realistic value? Our current tests use `r5d.2xlarge`